lib/rocket_pants/controller/respondable.rb

```diff
@@ -166,11 +164,7 @@ def exposes(object, options = {})
       elsif Respondable.collection?(object)
         collection object, options
       else
-        if Respondable.invalid?(object)
-          error! :invalid_resource, object.errors
```

Comment:
We don't depend on this behavior anywhere, do we?

Reply:
other way around -- we depend on rocket pants NOT having this behavior

Reply:
If we open source this project, will people appreciate this change or will e need to generalize it somehow?